src/pages/home/owners/OwnersTable.tsx

```diff
@@ -64,86 +68,102 @@ export default function OwnersTable({ ownersData }: OwnersTableProps) {
-                  count={ Math.ceil(ownersData.length / rowsPerPage) }
-                  onChange={ handlePageChange }
+                <Typography>INFO TBD</Typography>
+              </StyledTableCellHeader>
+            </TableRow>
+          </TableHead>
+          <TableBody sx={ { backgroundColor: theme.colors.grey600 } }>
+            { ownersData
+              .slice(
+                (page - 1) * rowsPerPage,
+                (page - 1) * rowsPerPage + rowsPerPage
+              )
+              .map((row) => (
+                <TableRow key={ row.id }>
+                  <StyledTableCell
+                    sx={ {
+                      color: theme.colors.white,
+                      maxWidth: ["150px", "unset"],
+                    } }
+                  >
+                    <Box
+                      sx={ { display: "flex" } }
+                      overflow="scroll"
+                      whiteSpace="nowrap"
+                    >
+                      <Box>{ row.name }</Box>
+                      { row.registered ? (
+                        <Box
+                          sx={ {
+                            marginLeft: "15px",
+                            display: { xs: "none", sm: "flex" },
+                          } }
+                        >
+                          <PendingBadge />
+                        </Box>
+                      ) : null }
+                    </Box>
+                  </StyledTableCell>
+                  <StyledTableCell
+                    sx={ {
+                      overflow: "scroll",
+                      whiteSpace: "nowrap",
+                      maxWidth: ["154px", "unset"],
+                    } }
+                  >
+                    { row.song }
+                  </StyledTableCell>
+                  <StyledTableCell
+                    sx={ { display: { xs: "none", sm: "table-cell" } } }
+                  >
+                    { row.info_tbd }%
+                  </StyledTableCell>
+                </TableRow>
+              )) }
+            { emptyRows > 0 && (
+              <TableRow style={ { height: rowHeight * emptyRows } }>
+                <StyledTableCell
+                  sx={ { backgroundColor: theme.colors.black100 } }
+                  colSpan={ 3 }
                 />
-              </Box>
-            </StyledTableCell>
-          </TableRow>
-        </TableFooter>
-      </Table>
-    </StyledTableContainer>
-  );
+              </TableRow>
+            ) }
+          </TableBody>
+          { ownersData.length > rowsPerPage ? (
+            <TablePagination
+              numberOfRows={ ownersData.length }
+              page={ page }
+              rowsPerPage={ rowsPerPage }
+              lastRowOnPage={ lastRowOnPage }
+              handlePageChange={ handlePageChange }
+              colSpan={ 3 }
+              rowItemName="owner"
+              paddingTop={ "12px" }
+            />
+          ) : (
+            ""
+          ) }
```

Comment:
I wonder if instead of returning a falsy empty string, we can return `null` so that it is explicit
```suggestion
          ) : 
          null }
```

Reply:
✅